### PR TITLE
Update RNA-seq to 3.0.0

### LINKED ISSUE #215: Zero length bam files
Pipeline often generates zero length bam files with no error and still generates expression levels making it impossible to run the QC pipeline as it requires the bam file.

Ran 35 fastq pairs with the results all under:

/pod/pstore/groups/treehouse/backed_up/archive/output/externalSamples/phs000699.v1.p1_OSGenomics

For the final 12 I backhauled the bam generated and of those 9 were zero length:

/pod/pstore/groups/treehouse/backed_up/archive/output/externalSamples/phs000699.v1.p1_OSGenomics

Full log output of these 12 is at:

/pod/pstore/groups/treehouse/backed_up/archive/output/externalSamples/phs000699.v1.p1_OSGenomics/log.txt

(run on 3 machines so output is interleaved with IP address as the prefix)

### PR DESCRIPTION
**Pull request by cket**
resolves #215

### CODE REVIEW

**Comment by cket** [comment]
Lets hold off on merging this until we can test it for the problems brought up in #215

**Review by fnothaft — APPROVED** [review]

File: rnaseq-cgl-pipeline/rnaseq-cgl-pipeline.cwl [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+outputs:
+  output_files:
+    type:
+      type: array
+      items: File
+    outputBinding:
+      glob: '*.tar.gz'
+    doc: "Result files RNA-seq CGL pipeline"
+
```
Comment (wshands) [inline rnaseq-cgl-pipeline/rnaseq-cgl-pipeline.cwl:140]: Hi CJ,
Please add these two extra output file arrays to the outputs: section of the CWl so the whole outputs: section will look like this:
outputs:
  output_files:
    type:
      type: array
      items: File
    outputBinding:
      glob: '*.tar.gz'
    doc: "Result files RNA-seq CGL pipeline"

  wiggle_files:
    type:
      type: array
      items: File
    outputBinding:
      glob: '*.wiggle.bg'
    doc: "Wiggle result files RNA-seq CGL pipeline"

  bam_files:
    type:
      type: array
      items: File
    outputBinding:
      glob: '*.bam'
    doc: "BAM result files RNA-seq CGL pipeline"

**Comment by fnothaft** [comment]
Just pinging, what's the status on this?

**Comment by jvivian** [comment]
Closing — zero-length bam issue resolved  and 3.0.x already released.

**Comment by cket** [comment]
@jvivian wrong PR? But we are actually going to close this PR and move it into the Toil-RNAseq, @fnothaft. I'm just waiting to finish testing it before opening the new PR & I wanted to keep the code public in the meantime

**Comment by fnothaft** [comment]
> @jvivian wrong PR? But we are actually going to close this PR and move it into the Toil-RNAseq

Ah, OK! So, the `toil-rnaseq` Dockerfile will live in Toil-RNAseq now? That SGTM.

**Comment by cket** [comment]
Continued in https://github.com/BD2KGenomics/toil-rnaseq/pull/33

**Pull request closed without merging**